scene/2d/tile_map_layer.h

```diff
@@ -527,6 +532,7 @@ class TileMapLayer : public Node2D {
 	// Not exposed to users.
 	TileMapCell get_cell(const Vector2i &p_coords) const;
 
+	static TileDrawRectParams compute_transformed_tile_dest_rect(const Vector2 &p_position, const Vector2 &p_dest_rect_size, const TileData *p_tile_data, int p_alternative_tile);
```

Comment:
I think would rather avoid declaring a new structure here. It makes it a bit more difficult to keep track of what should be used or not. You can use references instead, that's pretty common in the codebase:
```
static void compute_transformed_tile_dest_rect(rect2 r_dest_rect, bool r_transpose, const Vector2 &p_position, const Vector2 &p_dest_rect_size, const TileData *p_tile_data, int p_alternative_tile);
```

Reply:
Yeah I am not 100% sure, though I think defining a struct for 2 return values is pretty rare. My rule of thumb is likely to use a reference when you have, like, 2 of 3 values to return. For more, a struct better sense.

Reply:
Yeah, totally makes sense, felt iffy when adding that struct. Also one less "how to name this thing". 🙃